contracts/staking/TokenStaking.sol

```diff
@@ -931,6 +960,13 @@ contract TokenStaking is Ownable, IStaking {
         }
     }
 
+    /// @notice Delegate voting power from the stake associated to the
+    ///         `operator` to a `delegatee` address. Caller must be the owner
+    ///         of this stake.
+    function delegateVoting(address operator, address delegatee) external {
+        delegate(operator, delegatee);
+    }
```

Comment:
What if we rename `delegate` function in `T` token contract to `delegateVoting` to keep them aligned?

https://github.com/threshold-network/solidity-contracts/blob/fa83b98566b7c786a499fc438d97b4dd3eee0293/contracts/token/T.sol#L79

We could also rename the function in `Checkpoint` so that we do not need to have `function delegate(address operator, address delegatee)` a few lines below and just put this logic into `delegateVoting`.